README.md

```diff
@@ -1,2 +1,74 @@
-# angular-minification-safe-di-readme
-<p data-visibility='hidden'>View <a href='https://learn.co/lessons/angular-minification-safe-di-readme' title='angular-minification-safe-di-readme'>angular-minification-safe-di-readme</a> on Learn.co and start learning to code for free.</p>
+# Minification safe Dependency Injection
+
+## Overview
+
+Dependency Injection is great but we can bite ourselves in the foot if we aren't careful. Let's have a look at how dependency injection can fail, and what we do to make things sweet again.
+
+## Objectives
+
+- Describe Angular DI annotations
+- Use Array annotations syntax
+- Use $inject annotations syntax
+
+## Minification
+
+Minification is great - it reduces our code's file size and allows us to code wonderful things without impacting load times. However, it can cause a bit of an issue when mixed with dependency injection.
```

Comment:
You should talk a little bit more about minification here, as I don't know how familiar the students will be in this topic. Describe what minification is, then why it's useful. Maybe even show an example here of running code through a minifier when discussing this.